app/assets/scripts/map.js

```diff
@@ -74,10 +75,17 @@ function initLocationSearch(map) {
         fillOpacity: 1,
       },
     });
-    var placeDiv = document.createElement('div');
-    var br = document.createElement('br');
-    $(placeDiv).append(place.name, br, address);
-    infowindow.setContent(placeDiv);
+
+    // Show marker info bubble for searched location
+    var locationMarkerTemplate = $("#locationmarker-template").html();
```

Comment:
Could you choose slightly more descriptive names than `locationMarkerTemplate` and `markerTemplate`?